data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,288 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	(Take the train to the Saryd game.)`
+					goto saryd
+				`	(Take the train to the Kimek game.)`
+					goto kimek
+				`	(Take the train to the Arachi game.)`
+					goto arachi
+			
+			
+			label choice
+			branch "seen all"
+				has "coalition games: saryd"
+				has "coalition games: kimek"
+				has "coalition games: arachi"
+			
+			branch "seen saryd, kimek"
+				has "coalition games: saryd"
+				has "coalition games: kimek"
+			
+			branch "seen saryd, arachi"
+				has "coalition games: saryd"
+				has "coalition games: arachi"
+			
+			branch "seen only saryd"
+				has "coalition games: saryd"
+			
+			branch "seen kimek, arachi"
+				has "coalition games: kimek"
+				has "coalition games: arachi"
+			
+			branch "seen only kimek"
+				has "coalition games: kimek"
+			
+			branch "seen only arachi"
+				has "coalition games: arachi"
+			
+			label "seen only saryd"
+			choice
+				`	(Take the train to the Kimek game.)`
+					goto kimek
+				`	(Take the train to the Arachi game.)`
+					goto arachi
+				`	(Take the train to the cooperative game.)`
+					goto nwp
+			
+			label "seen only kimek"
+			choice
+				`	(Take the train to the Saryd game.)`
+					goto saryd
+				`	(Take the train to the Arachi game.)`
+					goto arachi
+				`	(Take the train to the cooperative game.)`
+					goto wp
+			
+			label "seen only arachi"
+			choice
+				`	(Take the train to the Saryd game.)`
+					goto saryd
+				`	(Take the train to the Kimek game.)`
+					goto kimek
+				`	(Take the train to the cooperative game.)`
+					goto nwp
+			
+			label "seen saryd, kimek"
+			choice
+				`	(Take the train to the Arachi game.)`
+					goto arachi
+				`	(Take the train to the cooperative game.)`
+					goto wp
+			
+			label "seen saryd, arachi"
+			choice
+				`	(Take the train to the Kimek game.)`
+					goto kimek
+				`	(Take the train to the cooperative game.)`
+					goto nwp
+			
+			label "seen kimek, arachi"
+			choice
+				`	(Take the train to the Saryd game.)`
+					goto saryd
+				`	(Take the train to the cooperative game.)`
+					goto wp
+			
+			label "seen all"
+			choice
+				`	(Take the last train.)`
+					goto wp
+				
+			
+			label saryd
+			apply
+				set "coalition games: saryd"
+			`	You arrive at a massive, stadium-like structure, only it's actually shaped like a half moon. The place overlooks a huge valley. Many small drones fly over it, transmitting live footage to the dozens of cinema-sized screens for those that can't get a clear view through the glass.`
+			`	You notice starting gates on one of the sides of the track. A few minutes pass before an alarm sounds, and the massive crowd roars. The doors open: several Saryd burst from the gates with wheeled machines, similar to chariots, attached to them. They are also carrying bows and arrows.`
```

Comment:
didn't do the announcers translation thing now because that'd need a branch and I suck at those, also didn't get any idea for what they'd say either way. it would probably be nice to add that, though.